nemo_aligner/utils/train_script_utils.py

```diff
@@ -95,6 +95,7 @@ def disable_data_callbacks(ptl_model, train_dataloader, train_ds):
 def init_using_ptl(ptl_trainer, ptl_model, train_dataloader, train_ds):
     """initializes the model using PTL
     """
+    ptl_model.setup_complete = True
```

Comment:
nit: can this be moved to the bottom of the init_using_ptl function?

Reply:
No we can't because it will cause an error when resume training. I move it to `init_peft` function, maybe that looks better? 
605bad8baf13e6f5107f9e330f79b269fb8866dd